picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+
+import os,sys
+if os.name == 'nt':
+    import win32api, win32con
+from datetime import datetime
+
+def Is_hidden(file_temp):
+    if os.name== 'nt':
+        attribute = win32api.GetFileAttributes(file_temp)
```

Comment:
Why are you using the win32 api? You can instead subclass QFileDialog and use Qt implemented methods. see http://doc.qt.io/qt-5/qfiledialog.html#Option-enum